cmd/podman/containers/cp.go

```diff
@@ -40,24 +55,316 @@ var (
 func cpFlags(cmd *cobra.Command) {
 	flags := cmd.Flags()
 	flags.BoolVar(&cpOpts.Extract, "extract", false, "Extract the tar file into the destination directory.")
-	flags.BoolVar(&cpOpts.Pause, "pause", true, "Pause the container while copying")
+	flags.BoolVar(&cpOpts.Pause, "pause", true, "Pause the container while copying (deprecated)")
+	_ = flags.MarkHidden("pause")
```

Comment:
Is there any reason why you did not use `flags.MarkDeprecated`

Reply:
I forgot its existence :)  If I need another push, I'll fix it :+1:

Reply:
Does MarkDeprecated hide the option?  I don't want users to see the option. Just want it to work so that there scripts would continue to work.

Reply:
Yes, that will hide the option but note that it's already hidden.

Reply:
Yes it is hidden. If you still use it the message you specify with `MarkDeprecated` is emitted to stderr.
See `podman image diff --archive alpine > /dev/null` for example.

Reply:
Actually, using `MarkDeprecated(...)` is pretty ugly as it prints the warning twice for rootless podman (due to re-exec):
```
podman (run-950) $ ./bin/podman cp --pause=false Makefile test:/
Flag --pause has been deprecated, pause has been deprecated and and has no effect
Flag --pause has been deprecated, pause has been deprecated and and has no effect
```